test/rpc_test.py

```diff
@@ -1260,6 +1260,29 @@ def test_local_shutdown(self):
         # pass in graceful=False to ensure that we don't wait for other workers.
         rpc.shutdown(graceful=False)
 
+    @dist_init
+    def test_debug_info(self):
+        # only test keys in this test case. Values should be covered by
+        # individual module debug info tests
+        from torch.distributed.rpc import (
+            _get_debug_info,
+            _rref_context_get_debug_info
+        )
+        from torch.distributed.rpc.api import _agent
+        import torch.distributed.autograd as dist_autograd
+
+        info = _get_debug_info()
+        rref_info = _rref_context_get_debug_info()
+        agent_info = _agent.get_debug_info()
```

Comment:
It looks like the `get_debug_info` of the `_agent` is not marked as private, is that intentional? I guess it doesn't matter too much since the whole `_get_debug_info` is private anyways.

Reply:
> I guess it doesn't matter too much since the whole _get_debug_info is private anyways.

Yes, both `_agent` and `_get_debug_info` are private, and shouldn't affect docs for now. Let me add another PR to make three get_debug_info API constant.